addons/website/__manifest__.py

```diff
@@ -275,14 +274,14 @@
         ],
         'website.assets_all_wysiwyg': [
             ('include', 'web_editor.assets_wysiwyg'),
-            ('include', 'web_editor.assets_legacy_wysiwyg'),
+            ('include', 'web_editor.assets_snippets_menu'),
             ('include', 'website.assets_wysiwyg'),
         ],
         'website.backend_assets_all_wysiwyg': [
             ('include', 'web_editor.backend_assets_wysiwyg'),
-            ('include', 'web_editor.assets_legacy_wysiwyg'),
+            ('include', 'web_editor.assets_snippets_menu'),
             ('include', 'website.assets_wysiwyg'),
-            'website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js',
+            'website/static/src/components/wysiwyg_adapter/*',
```

Comment:
Sure about this? I see there is `addons/website/static/src/components/wysiwyg_adapter/page_options.js` which would now be matched but you don't see to modify it in any way.